x-pack/metricbeat/magefile.go

```diff
@@ -154,7 +156,25 @@ func GoUnitTest(ctx context.Context) error {
 // Use RACE_DETECTOR=true to enable the race detector.
 func GoIntegTest(ctx context.Context) error {
 	return devtools.RunIntegTest("goIntegTest", func() error {
-		return devtools.GoTest(ctx, devtools.DefaultGoTestIntegrationArgs())
+		modulesFileInfo, err := ioutil.ReadDir("./module")
```

Comment:
should we do this change also for non-xpack metricbeat?

Reply:
I believe that at some point we may come to the same limit. Also, we can face any interferences between tests right now.

Reply:
also having both tested in the same way should reduce complexity. +1 to doing it now

Reply:
Change added.